test/backend/CMakeLists.txt

```diff
@@ -83,21 +83,32 @@ else()
   set(BACKEND_TEST_ARGS "")
 endif()
 
+include(setup_model_download.cmake)
+
 # ${ONNX_HOME} is the directory where onnx downloads real model files.
 # Model files are saved under ${ONNX_HOME}/models/model_name/model.onnx.
 # C/C++ and JNI tests run in parallel so they must use a different
 # ONNX_HOME to avoid conflicts.
+#
+# We now have our own model download targets and different backend tests
+# share the pre-downloaded model files. So we no longer need a different
+# ONNX_HOME for C/C++ and JNI tests.
```

Comment:
This is cool. Thanks!

Reply:
Thanks, so looking forward to have faster more reliable CIs. Big thanks @gongsu832